community-operators/cluster-logging/clusterlogging.v0.0.1.clusterserviceversion.yaml

```diff
@@ -13,6 +13,47 @@ metadata:
     containerImage: quay.io/openshift/cluster-logging-operator:latest
     createdAt: 2018-08-01T08:00:00Z
     support: AOS Logging
+    alm-examples: |-
+         [
+            {
+              "apiVersion": "logging.openshift.io/v1alpha1",
+              "kind": "ClusterLogging",
+              "metadata": {
+                "name": "instance"
+               },
+              "spec": {
+                "managementState": "Managed",
+                "logStore": {
+                  "type": "elasticsearch",
+                  "elasticsearch": {
+                    "nodeCount": 3,
+                    "redundancyPolicy": "SingleRedundancy",
+                    "storage": {
```

Comment:
we should default with the `storageClassName` here, the operator currently looks for both the storage class name and size

Reply:
done